src/military/military.cpp

```diff
@@ -6737,9 +6809,11 @@ void recover_org(sys::state& state) {
 		auto spending_level = in_nation.get_effective_naval_spending() * over_size_penalty;
 		auto modified_regen = regen_mod * spending_level / 150.0f;
 
+		auto max_org = 0.25f + 0.75f * spending_level + leader.get_prestige() * state.defines.leader_prestige_to_max_org_factor;
+
 		for(auto reg : ar.get_navy_membership()) {
 			auto c_org = reg.get_ship().get_org();
-			reg.get_ship().set_org(std::min(c_org + modified_regen, std::max(c_org, 0.25f + 0.75f * spending_level)));
+			reg.get_ship().set_org(std::min(c_org + modified_regen, std::max(c_org, max_org)));
```

Comment:
couldn't the leader prestige factor result in max org being > 1? that would break things. Also, I am not sure that this formula will work correctly. We scale org and strength to a 0 to 1 range. IIRC V2 does not (I know that it doesn't for strength; I forget how org works). Without taking the change in scale into account you may be altering org too much

Reply:
Fixed w #13ae911